Use "vendored-ssl" feature for paho-mqtt crate => passes the "vendored" option
to the openssl-sys crate

Index: Cargo.toml
--- Cargo.toml.orig
+++ Cargo.toml
@@ -80,7 +80,7 @@ sibyl = { version = "0.6.16", optional = true, feature
 ] }
 rdp-rs = { version = "0.1.0", optional = true }
 scylla = { version = "0.10.1", optional = true }
-paho-mqtt = { version = "0.13.3", optional = true }
+paho-mqtt = { version = "0.13.3", optional = true, features = ["vendored-ssl"] }
 csv = "1.3.0"
 pavao = { version = "0.2.12", optional = true }
 fast-socks5 = { version = "0.9.2", optional = true }
